src/vs/workbench/services/keybinding/browser/keybindingService.ts

```diff
@@ -269,6 +269,9 @@ export class WorkbenchKeybindingService extends AbstractKeybindingService {
 
 		// for standard keybindings
 		disposables.add(dom.addDisposableListener(window, dom.EventType.KEY_DOWN, (e: KeyboardEvent) => {
+			if (this._isInKeybindingHoldMode) {
+				return;
+			}
```

Comment:
If we start inline chat in listening mode and then close it clicking on x with the mouse, _isInKeybindingHoldMode is not set to true, breaking keybindings


https://github.com/microsoft/vscode/assets/16353531/c4c3b72c-d2f9-43c8-b057-d303eb98295c